lib/rest-middleware.js

```diff
@@ -1,11 +1,11 @@
-function restMiddleware(helmet, logger, restifyLinks) {
+module.exports = function restMiddleware(helmet, logger, restifyLinks) {
```

Comment:
Do we need this to day restMiddleware? Other files don't call out a function name in module.exports and we aren't calling the function specifically.